### PR TITLE
Makes radiation collectors buildable

### LINKED ISSUE #37128: Radiation collectors can't be constructed or deconstructed
What it says in the title: You can't deconstruct rad collectors or build new ones (no circuit boards exist either).

Also worth mentioning that they wrench/unwrench instantly unlike pretty much any other machinery.

### PR DESCRIPTION
**Pull request by DaxDupont**
:cl: Dax Dupont
add: You can now research and construct radiation collectors.
/:cl:

[why]: Fixes #37128

### CODE REVIEW

**Comment by 81Denton** [comment]
Very nice. My brain was too smooth to deal with the panel state/deconstruction part.

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 1]
```diff
@@ -534,6 +534,14 @@
 		/obj/item/stock_parts/capacitor = 1)
 	def_components = list(/obj/item/stock_parts/cell = /obj/item/stock_parts/cell/high/empty)
 
+/obj/item/circuitboard/machine/rad_collector
+	name = "Radiation Collector (Machine Board)"
+	build_path = /obj/machinery/power/rad_collector
+	req_components = list(
```
Comment (Jalleo) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: very cheap I would suggest it requires a matter bin and reinforced plasma glass too

**Review by Jalleo — APPROVED** [review]
Not easy to build from the get go unless people end up mapping other stuff in so fine with me.

**Comment by praisenarsie** [comment]
dude you need to make this more expensive or harder to get. you should know why.

**Comment by Jalleo** [comment]
@praisenarsie Trust me it was only a manipulator to begin with. Suggest the parts it should take. Maybe a few parts from telecomms.

**Comment by Michael-Ashfield** [comment]
I would suggest being added to the recipe:
- 1 Capacitor (To store the power it generates)
- 1 Matter bin (To store the plasma) 
- Reinforced plasma glass

Make it so it is abit harder to make so the CE cant fill the engine with rad collectors 2min into the round to double or triple power generation.

**Comment by 81Denton** [comment]
>Make it so it is abit harder to make so the CE cant fill the engine with rad collectors 2min into the round to double or triple power generation.

I don't think power generation on /tg/ is an issue when even the roundstart 6 collector setup on an N2 engine produces enough power for five stations.

**Comment by davethwave** [comment]
The radiation system fixes that problem if I remember

**Comment by DaxDupont** [comment]
@Michael-Ashfield i already did add that.

**Comment by ninjanomnom** [comment]
You'll be able to get a bit more power from stacking rad collectors but they inhibit the passing of radiation which reduces the amount of radiation that further out collectors can get. I doubt it'll be much of anything even on the second layer.

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 2]
```diff
@@ -534,6 +534,16 @@
...
+		/obj/item/stack/cable_coil = 5,
+		/obj/item/stock_parts/matter_bin = 1,
+		/obj/item/stack/sheet/plasmarglass = 1,
+		/obj/item/stock_parts/manipulator = 1)
```
Comment (ninjanomnom) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: I'm very hesitant to allow the construction of these at all but it doesn't make much sense that they can't be.

Err on the side of expensive for now, make it require a second glass and a capacitor.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**